src/index.js

```diff
@@ -319,18 +319,20 @@ class Dayjs {
     const that = dayjs(input)
     const zoneDelta = (that.utcOffset() - this.utcOffset()) * C.MILLISECONDS_A_MINUTE
     const diff = this - that
-    let result = Utils.m(this, that)
-
-    result = {
-      [C.Y]: result / 12,
-      [C.M]: result,
-      [C.Q]: result / 3,
-      [C.W]: (diff - zoneDelta) / C.MILLISECONDS_A_WEEK,
-      [C.D]: (diff - zoneDelta) / C.MILLISECONDS_A_DAY,
-      [C.H]: diff / C.MILLISECONDS_A_HOUR,
-      [C.MIN]: diff / C.MILLISECONDS_A_MINUTE,
-      [C.S]: diff / C.MILLISECONDS_A_SECOND
-    }[unit] || diff // milliseconds
+    const getMonth = () => Utils.m(this, that)
+
+    let result = {
+      [C.Y]: () => getMonth() / 12,
+      [C.M]: () => getMonth(),
+      [C.Q]: () => getMonth() / 3,
+      [C.W]: () => (diff - zoneDelta) / C.MILLISECONDS_A_WEEK,
+      [C.D]: () => (diff - zoneDelta) / C.MILLISECONDS_A_DAY,
+      [C.H]: () => diff / C.MILLISECONDS_A_HOUR,
+      [C.MIN]: () => diff / C.MILLISECONDS_A_MINUTE,
+      [C.S]: () => diff / C.MILLISECONDS_A_SECOND
+    }[unit] || (() => diff) // milliseconds
+
+    result = result()
```

Comment:
<img width="448" alt="image" src="https://user-images.githubusercontent.com/17680888/232309950-ee7596a3-6fcb-4dcc-bfdf-49cc65c2fc6e.png">
a lot of functions code will be added, which will increase the bundle size.
if we change this to `switch...case`, it will both improve performance, as well as minify the bundle code.

Reply:
done